esp/esp/program/modules/handlers/minorspolicymodule.py

```diff
@@ -38,15 +46,27 @@ def minorspolicyacknowledgement(self, request, tl, one, two, module, extra, prog
         context = {'prog': prog}
         if request.method == 'POST':
             context['form'] = minorspolicyacknowledgementform_factory(prog)(request.POST)
+            selection = context['form']['background_check']
```

Comment:
I think the reason your code wasn't working was here: `selection` is a `BoundField` rather than a string; you want `selection.data`.